### PR TITLE
add tables group

### PR DESCRIPTION
**Pull request by giovp**
refers to https://github.com/ome/ngff/pull/64/files#r1064614073

### CODE REVIEW

File: latest/index.bs [hunk 1]
```diff
@@ -190,283 +190,291 @@ OME-NGFF tables are compatible with the [AnnData model](https://github.com/scver
     |
     ├── .zattrs               # `.zattrs` of any group containing a table MUST contain `"tables"`, which is an array with the keys of the subgroubs that are a table.
```
Comment (giovp) [inline latest/index.bs]: can this be removed?

File: latest/index.bs [hunk 2]
```diff
@@ -190,283 +190,291 @@ OME-NGFF tables are compatible with the [AnnData model](https://github.com/scver
...
-        |           ├── .zarray  # `indptr` MUST be an `int` dtype.
-        |           |
-        |           ├── 0
-        │           |   ...
-        │           └── n
+        ├── .zattrs           # `.zattrs` MUST contain "tables", which lists the keys of the subgroups that are tables. In this case, the only table is "my_table".
+                              # hence `.zattrs` should be equal to `{ "tables": [ "table" ] }`.
         |
-        └── uns               # You MAY add a uns containter to store unstructured data.
+        └── table
```
Comment (will-moore) [inline latest/index.bs]: Should this match `"my_table"` above? And then also `{ "tables": [ "my_table" ] }` for the `.zattrs`.
I think that would make it clearer that it's a custom name

File: latest/index.bs (hunk 1 above)
Comment (will-moore) [reply latest/index.bs]: Yes, I think line 191 is wrong now. That will be `.zattrs` of the multiscales Image.

File: latest/index.bs (hunk 2 above)
Comment (giovp) [reply latest/index.bs]: that's a good point, I reverted back to `my_table` everywhere

**Comment by will-moore** [comment]
Looks good - you also want to fix line `191`?

**Comment by giovp** [comment]
hi @will-moore , yes, I've removed the comment but kept .zattr and .zgroup, let me know if they should be removed as well.
@kevinyamauchi I believe you'd have to merge this in order to then appear on your ngff PR, I don't have merge rights

**Review by will-moore — APPROVED** [review]
LGTM 👍

**Review by kevinyamauchi — APPROVED** [review]
LGTM! Thanks @giovp and @will-moore !

**Pull request merged**